src/database/factories/ComplaintFactory.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Complain;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Complain>
+ */
+class ComplaintFactory extends Factory
+{
+    /**
+     * Define the Complain model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition()
+    {
+        $created = User::factory()->create();
+        $related = User::factory()->create();
+
+        return [
+            'created_by' => $created->id,
+            'reviewed_by' => null,
+            'reviewed_at' => null,
+            'is_fixed' => 0,
+            'result' => $this->faker->text(100),
+            'subject' => $this->faker->text(100),
+            'description' => $this->faker->text(100),
+            'related_user_id' => $related->id,
+        ];
+    }
```

Comment:
Hocam database seeder içine  local için 30 40 tane  oluşturacak şekilde bi seeder oluşturup ekler misin?
arayüz giydirirken  pagination için vs  çok zaman kazandırıyor bunlar

![image](https://user-images.githubusercontent.com/6912824/223726515-bbe614c3-13dd-4e0d-8e9a-1da52252a16f.png)